modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -85,6 +87,25 @@ yarn remove @workday/canvas-kit-codemod
 > after executing the codemod, as its resulting formatting (spacing, quotes, etc.) may not match
 > your project conventions.
 
+## Component Updates
+
+### Styling API and CSS Tokens
+
+**PRs:** [#3101](https://github.com/Workday/canvas-kit/pull/3101)
+
+Several components have been refactored to use our new
+[Canvas Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) and
+[styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
+The React interface **has not changed**, but CSS variables are now used for dynamic properties.
```

Comment:
Do we want to reference "new" at this point? Or just say "refactored to use our [Canvas Tokens]

Reply:
we ca nix new